lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -369,6 +370,28 @@ getTransaction n = label "getTransaction" $ do
             value <- Coin <$> getWord64be
             return $ TxOut addr value
 
+--
+-- @
+-- FRAGMENT-ID = H(TYPE | CONTENT)
+-- CONTENT = #OUTPUTS (1 byte) | OUTPUT-1 | .. | OUTPUT-N
+-- OUTPUT = VALUE (8 bytes) | ADDR-SIZE (2 bytes) | ADDR (ADDR-SIZE bytes)
+-- TYPE = 1
+--
+-- H = blake2b_256
+-- @
+getLegacyTransaction :: Int -> Get Tx
+getLegacyTransaction size = do
+    bytes <- lookAhead (getLazyByteString $ fromIntegral size)
+    let tag = runPut (putWord8 (messageTypeTag MsgTypeLegacyUTxO))
```

Comment:
For the record, had the same thought here as said https://github.com/input-output-hk/cardano-wallet/pull/932#discussion_r340136412, with proposed change in #937.